ml_service/pipelines/diabetes_regression_build_train_pipeline_with_r.py

```diff
@@ -26,11 +26,16 @@ def main():
         print("aml_compute:")
         print(aml_compute)
 
-    # Create a run configuration environment
-    conda_deps_file = "diabetes_regression/training_dependencies.yml"
-    conda_deps = CondaDependencies(conda_deps_file)
-    run_config = RunConfiguration(conda_dependencies=conda_deps)
-    run_config.environment.docker.enabled = True
+    # Create a reusable run configuration environment
+    environment = Environment.load_from_directory(e.sources_directory_train)
```

Comment:
If this is going to create an environment from diabetes_regression/conda_dependencies.yml  then you will have to include r related dependencies in the cononda_dependencies.yml

Reply:
The doc explains that the r package must be uncommented. I've just fixed the path in the doc: https://github.com/microsoft/MLOpsPython/pull/168/commits/d4a589aabd62ecf5a2cac803e77cf11f01bdcd49
As it's quite big and slow I preferred to leave it out by default to make the first experience faster for the  majority of users who will be using Python.

Reply:
Also added comment in script https://github.com/microsoft/MLOpsPython/pull/168/commits/fe35f831ee0add4139f671dcba30e45e519e8e3b